### DISCUSSION TITLE
App is working in dev mode but after build run on simulation or real device app showing only white screen

### DISCUSSION THREAD

**Discussion by rkrajukhunt** [quasarframework/quasar#13764] [CLI - Capacitor / Cordova modes]
**when I am trying to run the app in dev mode in ios simulation, it's working fine
when I am trying to run the app in build mode app showing only a white screen on the simulation**

I am getting the error below, when I am running an app as build mode in simulation

objc[97463]: Class _PathPoint is implemented in both /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore (0x10ff1b340) and /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/TextInputUI.framework/TextInputUI (0x137b10fe8). One of the two will be used. Which one is undefined.

objc[97463]: Class _PointQueue is implemented in both /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore (0x10ff1b318) and /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/TextInputUI.framework/TextInputUI (0x137b11010). One of the two will be used. Which one is undefined.

**can anyone help to solve this error**

![image](https://user-images.githubusercontent.com/20268626/175287281-56bd2f33-215b-42b5-a792-e8fb7a3b6a69.png)

**Comment by serascandia** [comment]
in your capacitor.config.json in the server option if you see an ip address remove it and will work.

**Reply by NaturalDevCR** [reply]
> 
I checked on that but no ip address is there to remove, still I'm facing the exact same issue... blank screen on iOS... anything else coudl be the problem?

**Reply by geeksofttechnologies** [reply]
in your first screen you call a rest api or something from an url?
